test/ffi/js/test_ffi.re

```diff
@@ -243,7 +238,7 @@ let soundnessTests = Framework.suite("soundness", [
 
   [@ocaml.warning "-33"]
   test("resolved: Almost-Promise-like", () => {
-    let open Js_OO;
+    /* let open Js_OO; */
```

Comment:
I commented this out since didn't seem needed, but wanted to acknowledge that I have no clue what's Js_OO